src/main/scala/za/co/absa/ultet/model/table/TableDef.scala

```diff
@@ -141,10 +141,10 @@ object TableDef {
     }
 
     private def generateAlterForDescriptionChange(thisCol: TableColumn, otherCol: TableColumn): Seq[TableAlteration] = {
-      (thisCol.description, otherCol.description) match {
-        case (t, o) if t == o => Seq.empty // no change
-        case (Some(_), None) => Seq(TableColumnCommentDrop(tableIdentifier, thisCol.columnName))
-        case (_, Some(o)) => Seq(TableColumnCommentSet(tableIdentifier, otherCol.columnName, o)) // both add/set
+      if (thisCol.description != otherCol.description) {
+        Seq(TableColumnDescriptionSet(tableIdentifier, otherCol.columnName, otherCol.description)) // add/un-/set
+      } else {
+        Seq.empty // no change
```

Comment:
btw on line 154, in func `generateAlterForDefaultChange`, `case (Some(t), None)` -> `case (_, None)`